modules/kmm-applying-tolerations-to-kernel-module-pods.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="kmm-applying-tolerations-to-kernel-module-pods_{context}"]
+= Applying tolerations to kernel module pods 
+
+Taints and tolerations consist of `effect`, `key`, and `value` parameters. Tolerations include additional `operator` and `tolerationSeconds` parameters. 
+
+`effect`:: Indicates the taint effect to match. Empty means match all taint effects. When specified, allowed values are  `NoSchedule`, `PreferNoSchedule`, or `NoExecute`.
+
+`key`:: The taint key that the toleration applies to. Empty means match all taint keys. If the key is empty, `operator` must be `Exists`. This combination means to match all values and all keys.
+
+`value`:: The taint value the toleration matches to. If `operator` is `Exists`, the value should be empty, otherwise just a regular string.
+
+`operator`:: Represents a key's relationship to the value. Valid operators are `Exists` and the default value `Equal`. `Exists` is equivalent to wildcard for value, so that a pod can tolerate all taints of a particular category.
```

Comment:
```suggestion
`operator`:: Represents a key's relationship to the `value`. Valid `operators` parameters are: `Exists` and `Equal`. The default value is `Equal`. `Exists` is equivalent to wildcard for value, so that a pod can tolerate all taints of a particular category.
```

Reply:
If by "the value" you mean the same `value` that you have on line 15 in marks I'm not sure why you wouldn't.